server/cli/Cargo.toml

```diff
@@ -22,6 +22,8 @@ util = { path = "../util" }
 service = { path = "../service" }
 server = { path = "../server" }
 graphql = { path = "../graphql" }
+report_builder = {path = "../report_builder"}
```

Comment:
I've added report builder as a dependency rather than running the report builder binary.
I thought this made more sense? Though I know dependencies have caused slower build times so unsure if this is the right approach

Reply:
I think that's a good idea for now.
Might make sense to move more of this logic around I think, but need some more time to think about it...